src/api/index.ts

```diff
@@ -0,0 +1,2 @@
+export const getCommentsCount = (url: string) =>
```

Comment:
so, do we need other APIs? should they all be in the same place?

Reply:
Good point! Refactored it to the similar one from the `utils/api`